### ISSUE TITLE
[my-product] - Mstore Pro Expo payment problem

### ISSUE THREAD

**Issue by sajidgangat** [inspireui/support#2659]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue:
The payment gateway doesn't work. I tried multiple times and multiple ways. I will share the backend access to the WordPress site if you request. I have done everything as you have said. when I click on "Confirm Order" I get a blank page. I am really tired of this. If cannot work I will need a complete refund.
Currently, I am trying PayPal and it's not working

2. Screenshots issues (required):
![Uploading Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-11 at 10.01.48.png…]()

3. Proof of my purchased (required):
<img width="1200" alt="Screen Shot 2020-04-11 at 10 03 28 AM" src="https://user-images.githubusercontent.com/22694841/79045912-ba304000-7bdb-11ea-8b7f-d0ce06644494.png">

**Comment by minhcasi** [comment]
@sajidgangat - could you help to share the screenshot issue again, also attache the common folder so that we could reproduce the issue - thank you!

**Comment by sajidgangat** [comment]
sure I am sending the common folder and the screen shot. Whenever I select to pay then the webpage opens and it's empty. It does nothing. I cannot use any payment gateway except "cash on delivery".
Please find the attached document.
[common.zip](https://github.com/inspireui/support/files/4465822/common.zip)

1- When I click On Pay now I see this screen. 
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-11 at 23 47 55](https://user-images.githubusercontent.com/22694841/79060067-e46f1580-7c4e-11ea-956a-eeb8b17fa852.png)
2- After clicking "Pay Now", I am getting this page 
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-11 at 23 48 24](https://user-images.githubusercontent.com/22694841/79060081-036da780-7c4f-11ea-8528-855d5902ec35.png)

**Comment by sajidgangat** [comment]
If there is no update can I request a refund!! how long does the support take?
It's like no one cares here

**Comment by luyx2412** [comment]
Hi @sajidgangat Could you make sure to install the latest mstore-api - https://wordpress.org/plugins/mstore-api/ - and also using the guide from this link https://docs.inspireui.com/mstore/multi-payments/

**Comment by sajidgangat** [comment]
Yes, I tried That too but still, the problem exists.
Can You please help me set up the whole E-com Site of WordPress?
I can Provide my hosting Details if needed. In that way, you can check why exactly its not working.
Please provide your email so that I can send a fresh Cpanel so you can check and also configure and check if it works. Please make sure I am trying The MStore expo version.
Thanks

**Comment by sajidgangat** [comment]
if it connects to your application Then I will know that they might be a problem in the application that I am using. If its the WordPress problem you will know. Thanks
Please Provide an email so that I can allow access to WordPress. 
Regards

**Comment by sajidgangat** [comment]
Hi luyxtran264,
I have sent you all the info for the site to your email. please let me know once received.
The email that I sent to isl uyxtran264@gmail.com

**Comment by sajidgangat** [comment]
is there anyone who has time. I will surely leave a review on your product page about the support. There is no support timeline!!

**Comment by minhcasi** [comment]
@minhtruong315 - please help to check the api plugin issue - thank you!

**Comment by sajidgangat** [comment]
Thank you I will change the rating now. Please help me I am giving back 5 star

**Comment by minhtruong315** [comment]
@sajidgangat I'm adding NativeOnePageCheckout for mstore. I will send u updates when done. thanks

**Comment by sajidgangat** [comment]
sure please do let me know

**Comment by sajidgangat** [comment]
Hi I am just waiting on this. Please is there an update?

**Comment by minhtruong315** [comment]
I will send you updates today

**Comment by minhtruong315** [comment]
@sajidgangat are you using mstore expo version or cli version?

**Comment by sajidgangat** [comment]
> @sajidgangat are you using mstore expo version or cli version?

I am using the Expo version. React native

**Comment by minhtruong315** [comment]
[src.zip](https://github.com/inspireui/support/files/4498742/src.zip)
@sajidgangat replace these files
and change NativeOnePageCheckout:true in Config.js

**Comment by minhtruong315** [comment]
Hi, I'm closing this ticket as there is no response after more than 2 days, if you have any questions or reproducing the issue kindly reopen or create a new ticket - thank you!
